doc/overview.md

```diff
@@ -32,50 +32,58 @@ our [Discourse forum](https://forum.generic-mapping-tools.org/c/questions/pygmt-
 
 These are conference presentations about the development of PyGMT (previously "GMT/Python"):
 
+-   "Accessing and Integrating GMT with Python and the Scientific Python Ecosystem".
+    2024.
+	Yvonne Fröhlich, Dongdong Tian, Wei Ji Leong, Max Jones, and Michael Grund.
+	Presented at *AGU 2024*.
+	doi:[10.6084/m9.figshare.28049495](https://doi.org/10.6084/m9.figshare.28049495)
+
+    ![](https://s3-eu-west-1.amazonaws.com/pfigshare-u-previews/51279815/thumb.png){.align-center width="50%"}
```

Comment:
> The easiest solution is to upload the static images into the comment box, and then GitHub will store them for us, just like what we're doing in https://github.com/GenericMappingTools/pygmt/blob/main/doc/external_resources.md,

I use this trick all the time too, but I'm never sure how stable this actually is. What I'd recommend is to upload the full resolution thumbnail image to figshare (see e.g. how I uploaded a PNG thumbnail and PDF version of my [MIGARS poster](https://figshare.com/articles/poster/Applying_geometry-aware_Clifford_Fourier_Neural_Operators_to_weather_forecasting/25599978)) just in case. We can still use the GitHub link, but if that image ever disappears accidentally, we can retrieve it from figshare again.

Reply:
Maybe merge this PR and open a separat issue (and PR) for this?

Reply:
yes, we can discuss it later

Reply:
I opened issue to not forget about this #3708.